Makefile

```diff
@@ -643,8 +646,12 @@ push-release: package-version-to-tag ## Do an official release (Requires permiss
 
 .PHONY: release-images
 release-images: package-version-to-tag push catalog
+	$(RUNTIME) image tag $(OPERATOR_IMAGE) $(OPERATOR_TAG_BASE):latest
+	$(RUNTIME) image tag $(BUNDLE_IMG) $(BUNDLE_TAG_BASE):latest
+	$(RUNTIME) image tag $(CATALOG_IMG) $(CATALOG_TAG_BASE):latest
```

Comment:
This functionality is meant to be covered by the call below:

```
	# This will ensure that we also push to the latest tag
	$(MAKE) push TAG=latest
```

If that call is not sufficient, perhaps the `push` target needs to be fixed.

Reply:
So here's where it gets a little messy. At this point in release, we don't have `latest` of these images, we have the package-version-to-tag replaced ones (so they are `version`). These lines just re-tag the built images with latest, and `make push TAG=latest` is simply a push for those tags.
What I wanted to avoid was the need to give `push` a prereq on the build targets like we had before. It caused us to build separate images for version AND latest when we had this step. If you notice now, after the sdk update, `push` has no build prereqs.

Reply:
I see, that makes sense.